tests/live.rs

```diff
@@ -223,7 +283,8 @@ fn check_live_from_filterlists() {
 }
 
 #[test]
-fn check_live_deserialized_file() {
+#[ignore = "opt-in: requires BRAVE_SERVICE_KEY environment variable"]
+fn check_live_brave_deserialized_file() { // Note: CI relies on part of this function's name
```

Comment:
I agree that the key seems a bit contrived here; I had to add it a few months ago because the tests started failing due to unrelated serverside changes. I'll see if I can get a better answer for you from the devops team.